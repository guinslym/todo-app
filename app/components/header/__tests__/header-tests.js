import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Header from '../index';

describe('<Header/>', () => {
  let component;

  beforeEach(() => component = shallow(<Header/>));

  it('should render', () => {
    expect(component.length).toBeTruthy();
  });

  it('should display the header text', () => {
    expect(component.find('h1').text().trim()).toEqual('Todo App');
  });

});

describe('<Header/> snapshot', () => {

  it('should render correctly', () => {
    const tree = renderer.create(<Header/>).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
