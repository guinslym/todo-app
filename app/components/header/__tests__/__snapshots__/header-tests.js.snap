// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> snapshot should render correctly 1`] = `
<h1>
  Todo App
</h1>
`;
